the-blue-alliance-ios/View Controllers/Teams/Team/Team@Event/TeamAtEventViewController.swift

```diff
@@ -87,7 +86,7 @@ extension TeamAtEventViewController: MatchesViewControllerDelegate, TeamSummaryV
     }
 
     func matchSelected(_ match: Match) {
-        let matchViewController = MatchViewController(match: match, teamKey: teamKey, myTBA: myTBA, persistentContainer: persistentContainer, tbaKit: tbaKit, userDefaults: userDefaults)
```

Comment:
Oops missed this - we need to pass the teamKey still. This broke the bolding logic when testing.